server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -87,6 +87,70 @@ const virtualCardMutations = {
       return virtualCard;
     },
   },
+  createCardForCollective: {
+    description: 'Create new Stripe Virtual Card for existing hosted collective',
```

Comment:
I would make it generic and have a `provider` argument, it can default to Stripe. If it's different, simple exception.

Reply:
I really don't like writing code for maybe future logic.

Reply:
I still think it's more elegant and consistent, without requiring much code to be written.

As you want, that's fine.

Reply:
I have to disagree. We don't know if it will happen, and if it happen we don't know what we will do at this time. Maybe we will handle the provider as an .env variable, maybe it will be selected and associated with the collective elsewhere.
Problem is when someone will come back to this code in the future he might think the provider attribute is here for a reason and lose a lot of time figuring out why it is there.

Reply:
> As you want, that's fine.